lambda/main.tf

```
@@ -87,3 +87,10 @@ resource "aws_cloudwatch_query_definition" "lambda_statistics" {
     by bin(1h) as timeFrame
   QUERY
 }
+
+resource "aws_lambda_alias" "this" {
+  name             = var.alias_name
```

And then the check here to conditionally create the alias:
```suggestion
  count            = var.alias_name != "" ? 1 : 0
  name             = var.alias_name
```